lib/i18n/backend/active_record.rb

```diff
@@ -2,6 +2,7 @@
 
 require 'i18n/backend/base'
 require 'i18n/backend/active_record/translation'
+require 'pry-byebug'
```

Comment:
```suggestion
require 'i18n/backend/active_record/translation'
```

Reply:
ah my bad, forgot to remove my dev tooling